Privacy Portal Updates (#5621)
braze-inc/braze-docs: _docs/_user_guide/privacy_portal.md

---
nav_title: Privacy Portal
article_title: Privacy Portal
page_order: 60
layout: dev_guide
search_rank: 3
guide_top_header: "Braze Privacy Portal"

> guide_top_text: "> Welcome to the Braze Privacy Portal.  Here you’ll find all sorts of useful information about how Braze can help you be good custodians of your customers’ data and, importantly, enable you to take measures to comply with data protection rules relevant to your business.  We have brought together information and links to documentation that may assist you in your use of the Braze Services in compliance with applicable data protection laws and regulations."

Reviewer: ```suggestion
guide_top_text: "> Welcome to the Braze Privacy Portal. Here you’ll find useful information about how Braze can help you be good custodians of your customers’ data and, importantly, enable you to take measures to comply with data protection rules relevant to your business. We have brought together information and links to documentation that may assist you in your use of the Braze Services in compliance with applicable data protection laws and regulations."
```

> guide_top_text2: "When using Braze, Braze's customers are the data controllers who determine what data is collected and shared with the Braze Services. With respect to the Braze Services, Braze is a data processor that processes personal data on the instructions of our customers. Our customers are responsible for using the Braze Services in accordance with the data protection laws and regulations that apply to them. <br><br>As a good partner, we stay informed of the latest developments and practices in data protection and take action where required. For instance, we offer a Data Processing Addendum that covers necessary language for the EU and UK General Data Protection Regulation (GDPR), the California Consumer Privacy Act, as amended by the California Privacy Rights Act (CCPA), and other U.S. state privacy laws. The Braze Services include features that enable our customers to manage end-user consent, process “do not sell or share” requests, and action consumer rights such as deletion requests."

Reviewer: ```suggestion
guide_top_text2: "When using Braze, Braze customers are the data controllers who determine what data is collected and shared with the Braze Services. With respect to the Braze Services, Braze is a data processor that processes personal data on the instructions of our customers. Our customers are responsible for using the Braze Services in accordance with the data protection laws and regulations that apply to them. <br><br>As a good partner, we stay informed of the latest developments and practices in data protection and take action where required. For instance, we offer a Data Processing Addendum that covers necessary language for the EU and UK General Data Protection Regulation (GDPR), the California Consumer Privacy Act, as amended by the California Privacy Rights Act (CCPA), and other US state privacy laws. The Braze Services include features that enable our customers to manage end-user consent, process “do not sell or share” requests, and action consumer rights such as deletion requests."
```

> guide_top_text3: "<h4>The role of Braze Customers under data protection laws</h4><br>When using Braze, Braze’s customers are the data controllers who determine what data is collected and shared with the Braze Services. With respect to the Braze Services, Braze is a data processor that processes personal data on the instructions of our customers. Our customers are responsible for using the Braze Services in accordance with the data protection laws and regulations that apply to them."

Reviewer: ```suggestion
guide_top_text3: "<h4>The role of Braze customers under data protection laws</h4><br>When using Braze, Braze customers are the data controllers who determine what data is collected and shared with the Braze Services. With respect to the Braze Services, Braze is a data processor that processes personal data on the instructions of our customers. Our customers are responsible for using the Braze Services in accordance with the data protection laws and regulations that apply to them."
```

  fa_icon: fas fa-cogs
- name: Email Open Pixel and Click Tracking
  link: /docs/user_guide/data_and_analytics/tracking/email_tracking#open-pixel-and-click-tracking-overview
  fa_icon: fa-solid fa-envelope
- name: Canvas Audience Sync Privacy Considerations
  link: /docs/partners/canvas_steps/overview#data-privacy-considerations
  fa_icon: fa-solid fa-arrows-rotate

guide_menu_title: "Security"
guide_menu_list:
- name: Security, Privacy, and Architecture Datasheet
  link: https://www.braze.com/company/legal/security-schedule
  fa_icon: fa-solid fa-lock
- name: Braze Security Certifications
  link: /docs/developer_guide/disclosures/security_qualifications/#soc-2-examination
  fa_icon: fa-solid fa-lock

guide_menu_title2: "Legal"
guide_menu_list2:
- name: Braze Data Processing Addendum and FAQs
  link: https://www.braze.com/company/legal/dpa
  fa_icon: fa-solid fa-question
- name: Standard Contractual Clauses for Transfers from the UK and EU
  link: https://www.braze.com/company/legal/scc
  fa_icon: fa-solid fa-gavel
- name: Braze Subprocessors
  link: https://www.braze.com/company/legal/subprocessors
  fa_icon: fa-sharp fa-solid fa-network-wired

> - name: GDPR Frequently Asked Questions

Reviewer: ```suggestion
- name: GDPR FAQs
```
FAQs? Or maybe that's one too many acronyms back-to-back

  link: https://www.braze.com/company/legal/gdpr-faq
  fa_icon: fa-solid fa-question
- name: Acceptable Use Policy
  link: https://www.braze.com/company/legal/aup
  fa_icon: fa-solid fa-thumbs-up
- name: Innovation Statement
  link: /docs/developer_guide/disclosures/innovation_statement
  fa_icon: fa-regular fa-lightbulb

---

{% alert important %}

> **Legal Disclaimer**<br>

Reviewer: ```suggestion
**Legal Disclaimer:**<br>
```
